joomla/joomla-cms
# [5.0] TinyMCE highlighter plugin with CodeMirror 6

Pull Request for Issue #41278 .

### Summary of Changes

(Re)Writing a plugin for TinyMCE code highlighting with CodeMirror 6.


### Testing Instructions
Apply patch, run `npm install`.
Enable "code highlighting" in TinyMCE plugin parameters.
Open article editing and check that  "code highlighting" is working


### Actual result BEFORE applying this Pull Request
working


### Expected result AFTER applying this Pull Request
working


### Link to documentations
Please select:
